unity-renderer/Assets/DCLPlugins/ECS7/Systems/BillboardSystem/ECSBillboardSystem.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using DCL;
+using DCL.ECSComponents;
+using DCL.ECSRuntime;
+using DCL.Models;
+
+namespace ECSSystems.BillboardSystem
+{
+    public class ECSBillboardSystem : IDisposable
+    {
+        
+        private class State
+        {
+            public ECSComponent<PBBillboard> billboards;
+            public DataStore_Camera camera;
+        }
```

Comment:
Why are we wrapping `billboards` and `camera` on a class? it seems like an unnecessary allocation. We could have some readonly fields instead.